ui-v2/app/styles/components/healthchecked-resource/layout.scss

```diff
@@ -0,0 +1,78 @@
+%healthchecked-resource header span,
+%healthchecked-resource header em,
+%healthchecked-resource li:not(:last-child) span {
+  overflow: hidden;
+  display: inline-block;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  max-width: 100%;
+}
+%healthchecked-resource li:last-child:not(:only-child) {
+  overflow: hidden;
+  white-space: nowrap;
+}
+%healthchecked-resource,
+%healthchecked-resource li {
+  border-color: $ui-gray-200;
+}
+%healthchecked-resource,
+%healthchecked-resource header,
+%healthchecked-resource li {
+  position: relative;
+}
+%healthchecked-resource header strong {
+  position: absolute;
+}
+%healthchecked-resource a,
+%healthchecked-resource header a > * {
+  display: block;
+}
+%healthchecked-resource li::before {
+  left: 11px;
+  top: 50%;
+  margin-top: -0.49em !important;
+}
+.healthy .healthchecked-resource header span {
+  padding-right: 20px;
+}
+.healthy .healthchecked-resource li {
+  position: absolute;
+  top: 8px;
+  right: 16px;
+  border: none;
+}
+.healthy .healthchecked-resource li::before {
+  left: 0;
+}
+.healthy .healthchecked-resource li span {
+  display: none;
+}
+.healthy .healthchecked-resource li a {
+  padding-left: 0;
+}
+%healthchecked-resource header strong {
+  top: 2.8em;
+  left: 15px;
+}
+%healthchecked-resource header span {
+  margin-bottom: 1.75em;
+}
+%healthchecked-resource header a {
+  padding: 12px 15px;
+}
+.unhealthy .healthchecked-resource header a {
+  padding-bottom: 15px;
+}
+.unhealthy .healthchecked-resource header.with-service a {
+  padding-bottom: 25px;
+}
+%healthchecked-resource li a {
+  padding: 3px 15px;
+  padding-top: 4px;
+  padding-left: 39px;
+  height: 31px;
+}
+%healthchecked-resource li:not(.passing) strong,
+.healthy .healthchecked-resource li:only-child strong {
+  display: none;
+}
```

Comment:
This component seems to have an unusual amount of values that aren't coming from variables.

Reply:
Not every value needs to be a variable? Again, this isn't really part of this PR, all we've done here is moved files around and tweaked the color names (and fleshed out the variables)